### ISSUE TITLE
CPU testing worksfine, GPU gives random outputs

### ISSUE THREAD

**Issue by masoudMZB** [PaddlePaddle/PaddleOCR#10142]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：windows 10. cudatoolkit 10.2 . GPU = 3090 Ti
- 版本号/Version：Paddlepaddle-gpu：2.4.2 and 2.4.1 (both tested) PaddleOCR：2.6.0  问题相关组件/Related components：

- 运行指令/Command Code： python ./tools/infer_det.py -c "D:/Masoud/train_paddle/ch_PP-OCRv3_det_distill_train/ch_PP-OCRv3_det_cml.yml" -o Global.infer_img=C:/Users/masoudpa/Desktop/2.jpg Global.pretrained_model="D:/Masoud/train_paddle/ch_PP-OCRv3_det_distill_train/best_accuracy.pdparams" PostProcess.box_thresh=0.6 PostProcess.unclip_ratio=2.0 Global.use_gpu=False 
- python tools/infer/predict_det.py --image_dir=C:/Users/masoudpa/Desktop/chars_en_40.jpg --det_model_dir="D:/Masoud/train_paddle/ch_PP-OCRv3_det_infer/" --use_gpu=False --det_db_box_thresh=0.6 --det_db_unclip_ratio=1.5  
-  python tools/train.py -c "D:/Masoud/train_paddle/ch_PP-OCRv3_det_distill_train/ch_PP-OCRv3_det_cml.yml" -o Global.pretrained_model="D:/Masoud/train_paddle/ch_PP-OCRv3_det_distill_train/student.pdparams

- 完整报错/Complete Error Message：No error. just random output

As I mentioned while inferencing/testing when using only cpu ( with use_gpu=false) everything is fine. but when using GPU , outputs become all random and weird. I've tested many models with their corresponding config files:
- ch_ppocr_server_v2.0_det_infer
- ch_ppocr_server_v2.0_det_train
- ch_PP-OCRv3_det_distill_train
- ch_PP-OCRv3_det_infer
- det_mv3_db_v2.0_train
- det_r50_db++_icdar15_train
- det_r50_vd_db_v2.0_train
- en_PP-OCRv3_det_distill_train
- en_PP-OCRv3_det_infer
- Multilingual_PP-OCRv3_det_distill_train
- Multilingual_PP-OCRv3_det_infer

Also, training is not working too ( Both CPU and GPU have problems with training ). I used 1 image as a training image ( from the ICDAR dataset paddle recommended ) I used that image as the validation set too.  but after 100 epochs hmean is zero. I tested output pdparam to test the image, and nothing was detected there too.

**Comment by bahar3474** [comment]
I have the same issue.

**Comment by thebor1337** [comment]
the same issue

**Comment by masoudMZB** [comment]
Update : 
Upgrading to cudatoolkit 11.6 , cudnn = 8.4, and using paddlepaddle-gpu=2.4.1 helped in inferencing/testing but training is still in trouble.

Now my question is How to train? I repeated prev experiment and still heamn is 0 after 100 epochs

**Comment by vandesa003** [comment]
Same issue here. I am in CUDA 11.3. CPU works as expected but GPU provide random output.

**Comment by masoudMZB** [comment]
Try to update cuda cudnn and also check here : https://github.com/PaddlePaddle/PaddleOCR/issues/10194
